### PR TITLE
update setting ip_forwarding

### PR DESCRIPTION
**Pull request by mattkasun**
## Describe your changes
use os.Write on linux
added forwarding for windows

## Checklist before requesting a review
- [x] My changes affect only 10 files or less.
- [x] I have performed a self-review of my code and tested it.
- [ ] If it is a new feature, I have added thorough tests, my code is <= 1450 lines.
- [ ] If it is a bugfix, my code is <= 200 lines.
- [ ] My functions are <= 80 lines.
- [ ] I have had my code reviewed by a peer.
- [ ] My unit tests pass locally.
- [ ] Netclient & Netmaker are awesome.

### CODE REVIEW

File: nmproxy/router/iptables_linux.go [hunk 1]
```diff
@@ -248,12 +248,22 @@ func (i *iptablesManager) AddIngressRoutingRule(server, extPeerKey, extPeerAddr
 	if err != nil {
 		logger.Log(1, fmt.Sprintf("failed to add rule: %v, Err: %v ", ruleSpec, err.Error()))
 	}
+	ruleSpecForward := []string{"-i", "netmaker", "-j", "ACCEPT"}
+	err = iptablesClient.Insert(defaultIpTable, netmakerFilterChain, 1, ruleSpecForward...)
```
Comment (abhishek9686) [inline nmproxy/router/iptables_linux.go:252]: not the right place to add this rule
add it under `InsertEgressRoutingRules` and `InsertIngressRoutingRules`  funcs

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: nmproxy/router/iptables_linux.go (hunk 1 above)
Comment (mattkasun) [reply nmproxy/router/iptables_linux.go:252]: deleted changes.  will add to in a separate PR.

**Review by 0xdcarns — APPROVED** [review]

**Review by abhishek9686 — APPROVED** [review]

**Comment by 0xdcarns** [comment]
@theguy951357 please test windows on this one as well

**Review by theguy951357 — APPROVED** [review]
pr steps pass
critical list passes

**Pull request merged**